services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -40,3 +70,165 @@ async def service_retrieve_data_on_ports(
     )
     # validate and return
     return RetrieveDataOutEnveloped.parse_obj(resp.json())
+
+
+@router.post(
+    "/{node_uuid}:start",
+    summary="start the dynamic-sidecar for this service",
+    responses={
+        status.HTTP_500_INTERNAL_SERVER_ERROR: {
+            "description": "Error while starting dynamic sidecar"
+        }
+    },
+)
+async def start_dynamic_sidecar(
+    node_uuid: UUID, model: StartDynamicSidecarModel, app: FastAPI = Depends(get_app)
+) -> Dict[str, str]:
+    debug_message = (
+        f"DYNAMIC_SIDECAR: user_id={model.user_id}, project_id={model.project_id}, "
+        f"service_key={model.service_key}, service_tag={model.service_tag}, "
+        f"node_uuid={node_uuid}"
+    )
+    log.debug(debug_message)
```

Comment:
haha! I also like f-strings ... I think in py3.8 the logger will allow us to use them :-)

TIP: pydantic models ``__str__`` will display very similar to what you wrote, so it can be reduced to
```python
log.debug("DYNAMIC_SIDECAR: %s  node_uuid=%s", model, node_uuid")
```

Reply:
yes, Pydantic dose some nice formatting. I will accept your tip